Gemfile.lock

```diff
@@ -634,4 +636,4 @@ DEPENDENCIES
   websocket-client-simple!
 
 BUNDLED WITH
-   2.2.16
+   2.2.15
```

Comment:
@AdamMusa I particularly meant this whole file, can you remove these changes from the PR?